src/applications/claims-status/containers/AskVAPage.jsx

```diff
@@ -110,27 +110,27 @@ class AskVAPage extends React.Component {
                 </li>
                 <li>The date benefits will begin if we approve your claim</li>
               </ul>
-              <div className="usa-alert usa-alert-info background-color-only claims-alert">
```

Comment:
I'm guessing that removing this was also something discussed with design?

Reply:
Yes we discussed :)